src/core/linux/SDL_dbus.h

```diff
@@ -98,6 +98,8 @@ extern SDL_bool SDL_DBus_ScreensaverInhibit(SDL_bool inhibit);
 
 extern char *SDL_DBus_GetLocalMachineId(void);
 
+extern char **SDL_DBus_GetPortalFilePath(char *key, int *file_path_count);
```

Comment:
could this be renamed to SDL_DBus_GetPortalFilePaths because it returns multiple paths